src/index.js

```diff
@@ -1,17 +1,24 @@
 import neuralNetwork from "./NeuralNetwork";
 import handpose from "./Handpose";
 import poseDetection from "./PoseDetection";
+import bodyPix from "./BodySegmentation";
 import * as tf from "@tensorflow/tfjs";
 import * as tfvis from "@tensorflow/tfjs-vis";
 import p5Utils from "./utils/p5Utils";
+import preloadRegister from "./utils/p5PreloadHelper";
 
+const withPreload = {
+  bodyPix,
+}
 export default Object.assign(
   { p5Utils },
+  preloadRegister(withPreload),
```

Comment:
I might be mistaken, but previously libraries were either listed as "withPreload" or not - but not both.
If the preloadHelper isn't working as expected, best to not pull it in for now.